jsk_perception/src/point_pose_extractor.cpp

```diff
@@ -344,20 +344,20 @@ namespace jsk_perception
     server.setCallback(f);
 
     it = new image_transport::ImageTransport(*pnh_);
-    _client = pnh_->serviceClient<posedetection_msgs::Feature0DDetect>("/Feature0DDetect");
-    _pub = advertise<posedetection_msgs::ObjectDetection>(*pnh_, "/ObjectDetection", 10);
-    _pub_agg = advertise<posedetection_msgs::ObjectDetection>(*pnh_, "/ObjectDetection_agg", 10);
-    _pub_pose = advertise<geometry_msgs::PoseStamped>(*pnh_, "/object_pose", 10);
+    _client = nh_->serviceClient<posedetection_msgs::Feature0DDetect>("Feature0DDetect");
```

Comment:
Since this change is intended to preserve backward compatibility, it may be more helpful for future developers to add a comment like the following.
```suggestion
    // Use nh_ instead of pnh_ for backward compatibility.
    // See https://github.com/jsk-ros-pkg/jsk_recognition/pull/2779 and 
    // https://github.com/jsk-ros-pkg/jsk_recognition/pull/2778
    _client = nh_->serviceClient<posedetection_msgs::Feature0DDetect>("Feature0DDetect");
```